### PR TITLE
feat: add migration for explicit declare vite.config.js

### PR DESCRIPTION
**Pull request by Jordan-Hall**
fix: repair current dev vite excutor

### CODE REVIEW

File: packages/vite/src/executors/dev/executor.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ export default async function runExecutor(
         ? false
         : options.configFile
         ? joinPathFragments(`${context.root}/${options.configFile}`)
-        : false,
+        : undefined,
```
Comment (Jordan-Hall) [inline packages/vite/src/executors/dev/executor.ts:45]: @DominikPieper  this is the dev issue. Oversight on the change

Comment (DominikPieper) [reply packages/vite/src/executors/dev/executor.ts:45]: Okay great, thanks :-)

Comment (Jordan-Hall) [reply packages/vite/src/executors/dev/executor.ts:45]: Can you just check the migration. Never created a migration script for NX before

Comment (DominikPieper) [reply packages/vite/src/executors/dev/executor.ts:45]: Looks good to me. The most verbose stuff for migrations mostly is checking it's a project from the right plugin ^^'

**Pull request merged**